tests/ha/ha_cluster_join.pm

```diff
@@ -3,38 +3,38 @@
 # Copyright 2016-2018 SUSE LLC
 # SPDX-License-Identifier: FSFAP
 
-# Package: chrony ntp corosync-qdevice ha-cluster-bootstrap
+# Package: crmsh chrony ntp corosync-qdevice ha-cluster-bootstrap
 # Summary: Add node to existing cluster
 # Maintainer: QE-SAP <qe-sap@suse.de>, Loic Devulder <ldevulder@suse.com>
 
 use base 'opensusebasetest';
 use strict;
 use warnings;
-use testapi qw(is_serial_terminal :DEFAULT);
+use testapi;
 use lockapi;
+use serial_terminal qw(select_serial_terminal);
 use hacluster;
 use utils qw(zypper_call);
 
 sub wait_for_password_prompt {
     my %args = @_;
     $args{timeout} //= $default_timeout;
     $args{failok} //= 0;
-    die((caller(0))[3] . ' expects a needle tag or ref in the "needle => $tag" arg') unless $args{needle};
-    if (is_serial_terminal()) {
-        die "Timed out while waiting for password prompt" unless (wait_serial(qr/Password:\s*$/i));
-    }
-    elsif ($args{failok}) {
-        return (check_screen $args{needle}, $args{timeout});
-    }
-    else {
-        assert_screen $args{needle}, $args{timeout};
-    }
+    my $match = wait_serial(qr/Password:\s*$/i, timeout => $args{timeout});
+    return $match if ($args{failok});
+    die "Timed out while waiting for password prompt" unless ($match);
 }
 
 sub run {
     my $cluster_name = get_cluster_name;
     my $node_to_join = get_node_to_join;
 
+    select_serial_terminal;
+
+    # HA test modules use packages from ClusterTools2. Attempt to install it here and in
+    # ha_cluster_init, but continue if it's not possible (retval 104)
+    zypper_call('in ClusterTools2', exitcode => [0, 104]);
```

Comment:
```
  104 - ZYPPER_EXIT_INF_CAP_NOT_FOUND
           Returned by the install and the remove command in case any of the arguments does not match any of the
           available (or installed) package names or other capabilities.
```